arch/xtensa/include/esp32/efuse_reg.h

```diff
@@ -0,0 +1,1821 @@
+/****************************************************************************
+ * arch/xtensa/include/esp32/efuse_reg.h
```

Comment:
I don't think this file is in the right place.  Shouldn't be in arch/xtensa/src/esp32/hardware?